src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -9730,6 +9728,8 @@ DECLARE_API(GCWhere)
     return Status;
 }
 
+#ifndef FEATURE_PAL
+
 DECLARE_API(FindRoots)
 {
 #ifndef FEATURE_PAL
```

Comment:
Sorry, I wasn’t clear before but since your PR is ready now I do the CMakeFile.txt fix.